src/jimp.ts

```diff
@@ -1,4 +1,4 @@
-import jimp from 'jimp/es'
+import jimp = require('jimp')
```

Comment:
`/es` isn't actually ESM, it's still CJS. So loading the thing that doesn't try to pretend (too hard) it's ESM when it's not works out better 🙂